main.go

```diff
@@ -114,28 +128,42 @@ func main() {
 	flag.Parse()
 	buildinfo := buildinfo()
 	log.Printf("releasing with %s %s", *gobin, buildinfo)
-	var m manifest
-	for i, a := range assets {
-		assets[i].contents = readasset(a.builddir, a.goargs)
-	}
+
 	if slash := strings.IndexByte(*target, '/'); slash != -1 {
 		os, arch := (*target)[:slash], (*target)[slash+1:]
 		targets = append(targets[:0], tuple{os, arch})
 	}
-	for i := range targets {
-		for j := range tools {
+
+	for i := range dists {
+		dist := &dists[i]
+		dist.release()
+	}
+	for i := range dists {
+		if dists[i].sum == nil {
+			continue
+		}
+		log.Printf("manifest hash: SHA256:%x (%s)", dists[i].sum[:], buildinfo)
```

Comment:
This should probably contain the distribution name in it now that there could be multiple.  It would be unclear which one it's referring to otherwise, particularly when pasting for comparison.